packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -162,24 +162,25 @@ const getAppSlug = client => {
  * Initializes client OAuth workflow by storing the current information about
  * account type in localStorage. Generates the OAuth URL to stack endpoint,
  * passing the localStorage key as state in query string.
- * @param  {string} domain    Cozy domain
- * @param  {Object} konnector
- * @param  {string} redirectSlug The app we want to redirect the user on after the end of the flow
- * @param  {Object} extraParams some extra parameters to add to the query string
- * @param  {Boolean} reconnect Are we trying to reconnect an existing account ?
- * @param  {Boolean} manage Are we trying to manage an existing account ?
- * @param  {io.cozy.accounts} account targetted account if any
+ * @param  {Object} options
+ * @param  {CozyClient} options.client
+ * @param  {Object} options.konnector
+ * @param  {string} options.redirectSlug The app we want to redirect the user on after the end of the flow
+ * @param  {Object} options.extraParams some extra parameters to add to the query string
+ * @param  {Boolean} options.reconnect Are we trying to reconnect an existing account ?
+ * @param  {Boolean} options.manage Are we trying to manage an existing account ?
+ * @param  {CozyClientDocument} options.account targetted account if any
  * @return {Object}           Object containing: `oAuthUrl` (URL of cozy stack OAuth endpoint) and `oAuthStateKey` (localStorage key)
  */
-export const prepareOAuth = (
+export const prepareOAuth = ({
```

Comment:
Why this change?

Reply:
I think this is better to have named parameters with this number of parameters, plus this gets more consistent with getOAuthUrl.

Reply:
Done